sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,237 @@
+
+### sBTC Signer Responsibilities
+
+Overview of tasks the sBTC signers carry out:
+- Accept requests to deposit BTC.
+- Fulfill requests to withdraw BTC.
+- Complete deposit and withdrawal requests in a timely manner.
+- Maintain industry standard operational security around any hosts and private data (to include private keys).
+- Move BTC to a new UTXO when private keys are rotated.
+- Signers must perform UTXO consolidation as it is deposited [1].
+- Signers must deduct transaction fees from users in order to fund BTC withdrawal transactions:
+  - Ensure that the transaction fee is paid for (e.g., they deduct it from the user, and they set a minimum sBTC withdrawal amount).
+  - Transaction fee must be estimated proportionally for the requested operation [2].
+- Collectively, the signers must coordinate to calculate and advertise the fee parameters of the system:
+  - A minimum sBTC peg-out.
+  - STX transaction fee for minting the sBTC. This fee is paid by the user and can be sponsored by a 3rd party.
+
+### sBTC Signer Eligibility Criteria
+
+Signers will run the sBTC binary in addition to the core Stacks signer software and must meet certain criteria in order to facilitate the reliable functioning of the sBTC protocol at all times.
+
+The following eligibility criteria will be used to identify the sBTC Signers:
+
+- Does the proposed sBTC signer have a demonstrable operating history which shows their experience and reliability in running blockchain services?
+- Has the proposed sBTC signer participated in running a Stacks signer instance on Stacks 2.5 testnet or mainnet, and can they provide metrics showing this (ex: amount of stacks stacked over past several cycles)?
+  * Note: The sBTC signer is a Stacks event observer, meaning that the experience of running a Stacks node signer directly translates to running an sBTC signer.
+- Does the proposed sBTC signer agree to use reasonable efforts to maintain >99% uptime on the sBTC Signer? 
+  * Note: This metric may be self-affirmed if independent verification is not possible, or confirmed by on-chain voting/stacking activity.*
+- Does the proposed sBTC signer commit to a direct communication channel to be set up with the sBTC core engineers in order to respond to urgent updates within 24 hours?
+- Has the signer made contributions to Bitcoin or the Stacks network over the past year that demonstrate their commitment to the growth and success of the network? Examples include, but are not limited to: publishing independent research, marketing, co-authoring a SIP, submitting a Stacks pull request/issue, providing feedback on Stacks core development, or contributing to a Stacks Working Group.
+
+The criteria described above will be used to identify sBTC Signers that are able to meet some or all of the responsibilities described in the previous section.
```

Comment:
Might be worth mentioning in this section that the sBTC working group will be performing the vetting, instead of waiting until the `Activation` section to mention it.

Reply:
Also, are the sBTC developers part of the sBTC working group?  Is there a list of personnel somewhere who comprise the sBTC working group?

Reply:
Completed 3d680c1a2c2061a1c7ac8aa7622298e19b488621